cypress/fixtures/single_recipe.json

```diff
@@ -0,0 +1,11 @@
+{ 
+  "recipes":
```

Comment:
the response from the back-end is `recipe` not `recipes` when you return a single recipe.

Reply:
Our bad, resolved wit upcoming push.